src/HotChocolate/Core/src/Execution/DependencyInjection/InternalServiceCollectionExtensions.cs

```diff
@@ -166,8 +166,8 @@ internal static IServiceCollection TryAddDefaultCaches(
     {
         services.TryAddSingleton<IDocumentCache>(
             _ => new DefaultDocumentCache());
-        services.TryAddSingleton<IPreparedOperationCache>(
-            _ => new DefaultPreparedOperationCache());
+        services.TryAddTransient<PreparedOperationCacheOptions>(
```

Comment:
On the outer DI we only configure the options ... this means we have a central cache config. We can rethink that for the next major version but for 15 this is OK.

Reply:
Makes sense. I just wasn't sure if we need to continue support for people specifying a custom cache implementation.

Reply:
Ah right you can modify the schema services. I forgot about that, then it's no biggie.